cubism-core/src/moc.rs

```diff
@@ -123,7 +123,7 @@ impl Moc {
     /// Returns true if this model is masked.
     #[inline]
     pub fn is_masked(&self) -> bool {
-        self.drawable_masks.iter().any(|m| m.len() > 0)
+        self.drawable_masks.iter().any(|m| m.is_empty())
```

Comment:
this should be `self.drawable_masks.iter().any(|m| !m.is_empty())`

Reply:
Fixed on 8d85d3e.